### PR TITLE
Major Version Release v2.0.0

### LINKED ISSUE #7: Adding support for RuneMetrics API
Since this wrapper was created, the [RuneMetrics API](https://runescape.wikia.com/wiki/Application_programming_interface#Runemetrics) has come out. I was gonna work on trying to add that to the wrapper when I get a chance if that sounds useful?

### PR DESCRIPTION
**Pull request by pqt**
Closes #7

### CODE REVIEW

**Comment by pqt** [comment]
Just to update, the only remaining things are tests for the OSRS functions and content put onto the readme.

**Comment by pqt** [comment]
@Joshua-F my first question is whether or not to use the long or short forms for imports.

IE:
```js
import { ge } from "runescape-api"
// vs
import { grandexchange } from "runescape-api"
```

**Comment by 54** [comment]
add spedometer

File: src/runescape/bestiary.ts [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  SlayerCategory,
+  Weakness,
+} from "../lib/RuneScape"
+import { Jagex, Letter } from "../types"
+
+export const getAreas = async () => {
+  try {
+    const areas = await got(bestiary.endpoints.areas).json<
+      Jagex.Bestiary.Areas
+    >()
```
Comment (Joshua-F) [inline src/runescape/bestiary.ts]: Why have generics on a new line?

File: src/runescape/bestiary.ts [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  } catch (error) {
+    throw new Error(error)
+  }
+}
+
+export const getSlayerCategories = async () => {
+  try {
+    const categories = await got(bestiary.endpoints.slayerCategories).json<
+      Jagex.Bestiary.SlayerCategories
+    >()
```
Comment (Joshua-F) [inline src/runescape/bestiary.ts]: Same as above.

File: src/runescape/bestiary.ts [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+  } catch (error) {
+    throw new Error(error)
+  }
+}
+
+export const getWeaknesses = async () => {
+  try {
+    const weaknesses = await got(bestiary.endpoints.weaknesses).json<
+      Jagex.Bestiary.Weaknesses
+    >()
```
Comment (Joshua-F) [inline src/runescape/bestiary.ts]: ^^

File: src/runescape/bestiary.ts (hunk 1 above)
Comment (pqt) [reply src/runescape/bestiary.ts]: Prettier does the formatting on this one. Didn't give it any thought.

**Pull request merged**